app/setup/data/createLoadBalancerStep.go

```diff
@@ -74,22 +74,41 @@ func (fcs *createLoadBalancerStep) createServiceResource(ctx context.Context) er
 		},
 	}
 
-	// Delete for idempotence
-	err := fcs.clientSet.CoreV1().Services(fcs.namespace).Delete(ctx, serviceResource.Name, metav1.DeleteOptions{})
-	if err != nil && !errors.IsNotFound(err) {
-		return err
+	get, err := fcs.clientSet.CoreV1().Services(fcs.namespace).Get(ctx, serviceResource.Name, metav1.GetOptions{})
```

Comment:
Please improve the variable naming.
```suggestion
	runningLoadbalancerService, err := fcs.clientSet.CoreV1().Services(fcs.namespace).Get(ctx, serviceResource.Name, metav1.GetOptions{})
```